backend/app/libs/covalent/transfers.py

```diff
@@ -5,8 +5,9 @@
 from typing import Any, Generator, Optional
```

Comment:
In line with the refactoring I started, I suggest we move this module to `app/treasury/adapters`. Let's try to empty the `libs` folder as much as we can: adapters are always related to a given "domain" (treasury, spread), so we should make them live in the same place.

Note: we don't have yet modularized the pricing part (which is pretty independent of treasury), so there is no place where the coingecko adapter can settle for now. Let's just do things bit by bit.